src/org/zaproxy/zap/extension/codedx/UploadPropertiesDialog.java

```diff
@@ -68,6 +68,7 @@
 	private JTextField serverUrl;
 	private JTextField apiKey;
 	private JComboBox<NameValuePair> projectBox;
+	private JTextField timeout;
```

Comment:
Any reason not to use (for example) ZapNumberSpinner? This UI component just allows numbers and would avoid the need for conversions. (Just asking fine as is.)